src/requirements/__test__/exam-credit.test.ts

```diff
@@ -164,7 +164,6 @@ it('Exam is counted correctly for one major', () => {
     gradYear: '2020',
     entranceYear: '2016',
     college: 'EN',
-    program: '',
```

Comment:
To confirm: Why don't we include `program` here again?

Reply:
Two reasons. @vaishnavi17 used the prop 'grad' instead of 'program' so I switched to that in all my files. And then she also set the field as optional, so we don't need to include the string here anymore for the type check to pass